Feature/librosa exploration

nonfinito opened the pull request.

nonfinito committed:
demo + edit gitignore

nonfinito committed:
creation de spectogram.py qui génère un spectogram de l'audio et l'enregistre dans le dossier data/images

nonfinito committed:
ameliration de spectogram.py avec Path/pathlib

Anatole-DC committed:
Merge branch 'master' into feature/librosa-exploration

Anatole-DC commented on koregraph/api/spectogram.py:
```diff
@@ -0,0 +1,34 @@
+        Path: path to the saved image
+    '''
+
+    image_folder = os.environ.get("IMAGE_FOLDER")
+    if not os.path.exists(image_folder):
+        os.makedirs(image_folder)
```
- [x] Extract to global environment variable
- [x] Use `pathlib.Path` `mkdir(parents=True, exist_ok=True)` to check for existance

nonfinito committed:
Extract to global environment variable --> OK
Use pathlib.Path mkdir(parents=True, exist_ok=True) to check for existance --> OK

nonfinito committed:
Merge branch 'feature/librosa-exploration' of github.com:Anatole-DC/koregraph into feature/librosa-exploration

nonfinito committed:
Extract to global environment variable ok +  Use pathlib.Path ok

nonfinito committed:
ajout de prepare dataset + poetry run black sur spectogram
